MERP_Character_Sheet_BE/Repository/CharacterRepository.cs

```diff
@@ -0,0 +1,94 @@
+﻿using MERP_Character_Sheet_BE.Interfaces;
```

Comment:
This is just an example, but this could be `CharacterMemoryRepository` and on the Service we can use `CharacterMemoryRepository` and `CharacterDBRepository` with Mongo or whatever implementation